src/index.js

```diff
@@ -156,6 +160,7 @@ async function run() {
 
       if (skipEmptyRelease && cleanChangelog === '') {
         core.info('Generated changelog is empty and skip-on-empty has been activated so we skip this step')
+        core.setOutput('version', oldVersion)
```

Comment:
Can you also define this output in the `action.yml`

Reply:
It is already there.  I just make sure it always has a value.   The "version" is empty when no change is detected.  This makes downstream actions that rely on the version to have a value to fail.

Reply:
I will work on the test case for skip-tag

Reply:
Sorry, read incorrectly, thought you where setting `old-version`